sql/updates/latest-dev.sql

```diff
@@ -47,3 +47,75 @@ LANGUAGE C VOLATILE;
 DROP PROCEDURE IF EXISTS _timescaledb_functions.policy_compression_execute(job_id INTEGER, htid INTEGER, lag ANYELEMENT, maxchunks INTEGER, verbose_log BOOLEAN, recompress_enabled  BOOLEAN, use_creation_time BOOLEAN);
 
 DROP PROCEDURE IF EXISTS _timescaledb_functions.policy_compression(job_id INTEGER, config JSONB);
+
+CREATE PROCEDURE @extschema@.convert_to_columnstore(
+    uncompressed_chunk REGCLASS,
```

Comment:
These procedures should match the procedures in maintenance_util.sql. Check all parameter names.

Reply:
Fixed these.